src/tv2_afvd_showstyle/GlobalAdlibActionsGenerator.ts

```diff
@@ -351,6 +353,25 @@ export class GlobalAdlibActionsGenerator {
 		}
 	}
 
+	private makeFadeSoundPlayerAction(): IBlueprintActionManifest {
+		const fadeDownSoundPlayer: ActionFadeDownSoundPlayer = {
+			type: AdlibActionType.FADE_DOWN_SOUND_PLAYER
+		}
+		return {
+			externalId: generateExternalId(this.context.core, fadeDownSoundPlayer),
+			actionId: AdlibActionType.FADE_DOWN_SOUND_PLAYER,
+			userData: fadeDownSoundPlayer,
+			userDataManifest: {},
+			display: {
+				_rank: 400,
+				label: t(`Fade down sound player`),
+				sourceLayerId: SourceLayer.PgmAudioBed,
+				outputLayerId: SharedOutputLayer.SEC,
```

Comment:
I don't think it changes much because this is not a piece, but for consistency I would use the `SharedOutputLayer.MUSIK` output layer

Reply:
I would too. It has been updated